action-roku.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python2
+# -*-: coding utf-8 -*-
+
+import ConfigParser
+from hermes_python.hermes import Hermes
+import io
+from snipsroku.snipsroku import SnipsRoku
+
+CONFIGURATION_ENCODING_FORMAT = "utf-8"
+CONFIG_INI = "config.ini"
+
+MQTT_IP_ADDR = "localhost"
+MQTT_PORT = 1883
+MQTT_ADDR = "{}:{}".format(MQTT_IP_ADDR, str(MQTT_PORT))
+
+class SnipsConfigParser(ConfigParser.SafeConfigParser):
+    def to_dict(self):
+        return {section: {option_name : option for option_name, option in self.items(section)} for section in self.sections()}
```

Comment:
Maybe this line could be split into several ? Not really readable as is...

Reply:
It a function from the app template